firmware/application/apps/ui_ss_viewer.hpp

```diff
@@ -31,6 +31,8 @@
 
 namespace ui {
 
+static const std::filesystem::path splash_dot_bmp{u"/splash.bmp"};
```

Comment:
I think you'll want this to be constexpr or move the definition to a cpp file and forward declare an extern otherwise you'll end up with an ODR violation if something else includes this header.

Reply:
Actually, I lied. You won't get an ODR, each TU will instead get a copy. Maybe the linker will collapse?
Still probably better to just move to the cpp file unless you really need to export it.

Reply:
Updated